app/controllers/application_controller.rb

```diff
@@ -1,44 +1,39 @@
 class ApplicationController < ActionController::Base
-  # Adds a few additional behaviors into the application controller 
+  # Adds a few additional behaviors into the application controller
    include Blacklight::Controller
-  # Please be sure to impelement current_user and user_session. Blacklight depends on 
-  # these methods in order to perform user specific actions. 
```

Comment:
These whitespace changes make it more difficult to understand what's actually changing in this commit.